R/buildsteps_templates.R

```diff
@@ -510,6 +512,8 @@ cr_buildstep_r <- function(r,
      name <- basename(name)
   }
 
+  r_cmd = match.arg(r_cmd)
```

Comment:
This should be <-  :)

Reply:
Ahh! I changed now - next PR will have it.